components/elm/bld/namelist_files/namelist_definition.xml

```diff
@@ -359,6 +359,62 @@ Toggle to turn on FATES no competition mode (only relevant if FATES is being use
 Toggle to turn on FATES satellite phenology mode (only relevant if FATES is being used).
 </entry>
 
+<entry id="fates_radiation_model" type="char*256" category="physics"
+        group="elm_inparm" valid_values="norman, twostream" value="norman">
```

Comment:
There can't be any spaces between the valid values - e.g. this should be "norman,twostream"

Reply:
I just ran into this as well, and note that it applies to all the ones below too